x11-packages/rnote/build.sh

```diff
@@ -0,0 +1,67 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/flxzt/rnote"
+TERMUX_PKG_DESCRIPTION="An infinite canvas vector-based drawing application for handwritten notes"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="0.11.0"
+TERMUX_PKG_SRCURL="https://github.com/flxzt/rnote/archive/v${TERMUX_PKG_VERSION}/v${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SHA256=b133d4331963d3c09d3a7477f60fc4c5072471dcbf459379a593ca1724164af4
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_DEPENDS="gdk-pixbuf, gettext, glib, graphene, gtk4, hicolor-icon-theme, libadwaita, libcairo, pipewire, pango, poppler"
+TERMUX_PKG_BUILD_DEPENDS="libiconv"
+
+__fetch_gettext_rs() {
+    # This version is relative to gettext-sys provided by the gettext-rs crate
+	local _version="0.21.4"
+	local _url="https://github.com/gettext-rs/gettext-rs/archive/refs/tags/gettext-sys-$_version.tar.gz"
+	local _path="$TERMUX_PKG_CACHEDIR/gettext-v$_version.tar.gz"
+	termux_download "$_url" "$_path" SKIP_CHECKSUM
+	tar xf "$_path" -C "$TERMUX_PKG_SRCDIR"
+	mv "gettext-rs-gettext-sys-$_version" "gettext-source"
+
+
+	# Fetch latest and greatest gettext from upstream and place it inside gettext-source/gettext-sys
+	local _upstream_source_version="0.22.5"
+	wget "https://mirrors.kernel.org/gnu/gettext/gettext-${_upstream_source_version}.tar.xz" -P "$TERMUX_PKG_SRCDIR/gettext-source/gettext-sys"
+}
+
+__patch_gettext_rs() {
+    # Patch gettext-rs crate to use a newer gettext version because the old one doesn't compile properly with clang
+    patch -p1 -d "$TERMUX_PKG_SRCDIR/gettext-source" < "$TERMUX_PKG_BUILDER_DIR"/gettext-rs-crate-patch.diff
```

Comment:
Use `-i` to specify the input patch file instead of redirecting it in.

Bash would be an example for this usage.
https://github.com/termux/termux-packages/blob/702197226e77dc93ed2eb91b04e519f7b6bb5862/packages/bash/build.sh#L72-L81
Though in that case we are downloading upstream patches, since that's how Bash does bugfix releases.